cdap-examples/Purchase/README.md

```diff
@@ -5,17 +5,17 @@ and makes it available via ad-hoc querying and the RESTful interface of a Servic
 
 Features introduced: Custom Dataset with ad-hoc querying capability, Workflow, and MapReduce.
 
-- Uses scheduled MapReduce Workflows to read from one ObjectStore Dataset
+- Uses a scheduled MapReduce program and Workflow to read from one ObjectStore Dataset
```

Comment:
Workflow contains one or more MapReduce programs. And one schedules a Workflow. So I am not sure scheduled MapReduce program and Workflow is right. Alternative : "Uses a scheduled Workflow which starts a MapReduce to read from..."